### PR TITLE
Switch to Jenkins Pipelines

### PR DESCRIPTION
**Pull request by mortalisk**

### CODE REVIEW

File: testjenkins.sh [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+clone_repos () {
+	echo "Cloning into $LIBRES_ROOT"
+	git clone . $LIBRES_ROOT
+
+	source ./.libecl_version
+
+	echo "Cloning into $LIBECL_ROOT"
+	git clone https://github.com/equinor/libecl $LIBECL_ROOT
+	pushd $LIBECL_ROOT
+	git checkout tags/$LIBECL_VERSION
```
Comment (oyvindeide) [inline testjenkins.sh]: Can consider just checkout out the specific tag, for example like this:
git clone -b $LIBECL_VERSION --single-branch https://github.com/equinor/libecl $LIBECL_ROOT

File: testjenkins.sh [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+create_directories () {
+	mkdir $INSTALL
+	mkdir $LIBECL_BUILD
+	mkdir $LIBRES_BUILD
+}
+
+clone_repos () {
+	echo "Cloning into $LIBRES_ROOT"
+	git clone . $LIBRES_ROOT
```
Comment (oyvindeide) [inline testjenkins.sh:121]: Will this disregard local changes? Shouldn't be relevant for Jenkins though.

Comment (mortalisk) [reply testjenkins.sh:121]: Yes. I will add a comment to clarify that in the top of the script

File: python/res/job_queue/ext_job.py [hunk 3]
```diff
@@ -76,7 +76,10 @@ def __init__(self, config_file, private, name = None , license_root_path = None
             raise IOError('No such config file "%s".' % config_file)
 
     def __repr__(self):
-        return self._create_repr('%s, config_file = %s' % (self.name() , self.get_config_file()))
+        if self._address():
+            return self._create_repr('%s, config_file = %s' % (self.name() , self.get_config_file()))
```
Comment (oyvindeide) [inline python/res/job_queue/ext_job.py]: Since you're already changing this, you could change to .format instead of %, but not very important.

**Review by oyvindeide — APPROVED** [review]
Could perhaps add a more extensive commit message, but otherwise very good :+1:

**Pull request merged**